odoo_tools/tasks/main.py

```diff
@@ -1,11 +1,12 @@
 from invoke import Collection, Program
 
-from . import database, lastpass, module, pr, translate
+from . import database, lastpass, migrate, module, pr, translate
 
 tasks_ns = Collection()
 
 tasks_ns.add_collection(database)
 tasks_ns.add_collection(lastpass)
+tasks_ns.add_collection(migrate)
```

Comment:
New features should be added as new cli tools, not tasks.

Reply:
Yes that's what I guessed by looking here and there, but it was the first step to get something working ASAP by copying the former invoke task.